kod/object/item/passitem/defmod/helmet/mask/dratmask.kod

```diff
@@ -0,0 +1,44 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+DuskRatMask is FaceMask
+
+constants:
+
+   include blakston.khd
+
+resources:
+
+   DuskRatMask_name_rsc = "dusk rat mask"
+   DuskRatMask_icon_male_rsc = mmdrat.bgf
+   DuskRatMask_icon_female_rsc = mfdrat.bgf
+   DuskRatMask_desc_rsc = \
+      "A mask with the silo wet  of the poisonousness dusk rat.  It doesn't seem to make "
```

Comment:
Have some issues with the mask description... "silo wet" should be "silhouette"; remove the extra space there and change "poisonousness" to poisonous (technically the rat would be venomous but our usage of poison has already been wrong for 20 years so no point changing it now).